MySqlFiles/HighSchool.sql

```diff
@@ -0,0 +1,47 @@
+CREATE DATABASE high_school;
+
+CREATE TABLE IF NOT EXISTS student(
+	registration_number int NOT NULL PRIMARY KEY,
+    dni_number char(8) UNIQUE NOT NULL,
+    first_name varchar(255) NOT NULL,
+    last_name varchar(255) NOT NULL,	
+    birth_date date
+);
+
+CREATE TABLE IF NOT EXISTS teacher(
+	teacher_id int NOT NULL PRIMARY KEY,
+    first_name varchar(255) NOT NULL,
+    last_name varchar(255) NOT NULL,	
+    birth_date date
+);
+
+CREATE TABLE IF NOT EXISTS course(
+	course_id int NOT NULL PRIMARY KEY,
+    course_name varchar(255) NOT NULL,
+    assigned_teacher varchar(255),
+	hours_by_week time,   
+    CONSTRAINT assigned_teacher_fk FOREIGN KEY(assigned_teacher)
+    REFERENCES teacher(teacher_id),
+    CONSTRAINT students_fk FOREIGN KEY(assigned_course)
```

Comment:
Check this foreign key, if you are trying to relate students with courses you should first analyze how many students are taking one course and how many courses is taking one student. Then you can model this using the right approach. Let me know the answers to prior questions please